scripts/imagebuild.sh

```diff
@@ -28,29 +28,26 @@ cd "$base_dir" || {
 }
 cd ${base_dir}
 
-export REGISTRY=quay.io/
 
-echo "Copy apiserver"
+if [[ "${TRAVIS_PULL_REQUEST}" == "false" ]]; then
```

Comment:
I would say if branch != master and then early exit - that way the rest of the file isn't under a level of indent.